script.js

```diff
@@ -0,0 +1,49 @@
+//ЗАДАЧА 1
+
+let arr = [
+  [1],
+  [3, 5],
+  [7, 9, 11],
+  [13, 15, 17, 19],
+  [21, 23, 25, 27, 29]
+];
+
+function rowSumOddNumbers(i) {
```

Comment:
Nice try, but the function should work for any `i`, not just 1 - 3. :smiling_imp: 

By the way, 
```
rowSumOddNumbers(0)
script.js:23 Uncaught TypeError: Cannot read property 'length' of undefined
    at rowSumOddNumbers (script.js:23)
    at <anonymous>:1:1
```